nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/listener/CommentNotificationListener.java

```diff
@@ -37,13 +38,10 @@ public void handleNotifications(Event event) {
         EventContext ctx = event.getContext();
         DocumentEventContext docCtx = (DocumentEventContext) ctx;
         if (docCtx.getSourceDocument().getType().equals("Post")
+                || docCtx.getSourceDocument().getType().equals(AnnotationConstants.ANNOTATION_DOC_TYPE)
                 || docCtx.getSourceDocument().getType().equals(CommentsConstants.COMMENT_DOC_TYPE)) {
-            CommentManager commentManager = Framework.getService(CommentManager.class);
-            DocumentModel thread = commentManager.getThreadForComment(docCtx.getSourceDocument());
-            if (thread != null) {
-                Object[] args = { thread, null };
-                docCtx.setArgs(args);
-            }
+            Object[] args = { docCtx.getProperty(CommentConstants.TOP_LEVEL_DOCUMENT) };
+            docCtx.setArgs(args);
```

Comment:
Don't you need to keep the second value (the `null` one) in the array? it seems required as we did it previously and in the class constructor.